examples/jenkins/pipeline/bluegreen-pipeline.yaml

```diff
@@ -405,6 +405,21 @@ parameters:
   name: NAME
   required: true
   value: nodejs-postgresql-example
+- description: The OpenShift Namespace where the NodeJS and postgresql ImageStreams reside.
+  displayName: Namespace
+  name: NAMESPACE
+  required: true
+  value: openshift
+- description: Version of NodeJS image to be used (14-ubi8, or latest).
+  displayName: Version of NodeJS Image
+  name: NODEJS_VERSION
+  required: true
+  value: 14-ubi8
+- description: Version of PostgreSQL image to be used (12-el8, or latest).
+  displayName: Version of PostgreSQL Image
+  name: POSTGRESQL_VERSION
+  required: true
+  value: 12-el8
```

Comment:
I don't see this parameter being used anywhere in the template.

Reply:
https://github.com/openshift/origin/blob/master/examples/jenkins/pipeline/bluegreen-pipeline.yaml#L398